Library/Formula/sqlite.rb

```diff
@@ -36,15 +30,11 @@ class Sqlite < Formula
   end
 
   resource "docs" do
-    url "https://sqlite.org/2016/sqlite-doc-3150000.zip"
-    version "3.15.0"
+    url "https://www.sqlite.org/2023/sqlite-doc-3420000.zip"
+    version "3.42.0"
     sha256 "cea0208b2b8dddf51b846d6e9d23210fc497da2c6f3fbed96f00eadceb1c32f7"
```

Comment:
Looks like the docs checksum got missed?

Reply:
Oops, sorry. Let me fix that and push the change in a moment.